sass/gallery.scss

```diff
@@ -47,4 +47,61 @@ p {
 .images img {
     width: 100%;
     height: 285px;
+    cursor: pointer;
+}
+
+#modal-background {
+    background-color: #000000dd;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: 1;
+    display: none;
+    cursor: pointer;
+}
+
+#modal {
+    text-align: center;
+    background-color: white;
+    border-radius: 20px;
+    padding: 20px;
+    top: 50px;
+    left: 15%;
+    width: 70%;
+    position: absolute;
+    z-index: 2;
+    display: none;
+
+    i, p {
+        color: #F98728;
+        font-size: 20px;
+    }
+
+    img {
+        width: calc(100% - 60px);
+        margin: 30px 20px 20px 20px;
+    }
+}
+
+#close-modal {
+    width: calc(100% - 40px);
+    display: block;
+    text-align: right;
+    position: absolute;
+    top: 20;
```

Comment:
Firefox complains about these for some reason
![image](https://user-images.githubusercontent.com/2782749/105132764-7434d080-5ab9-11eb-8bab-651d7fc78b99.png)